truss/templates/server/truss_server.py

```diff
@@ -66,33 +66,25 @@ def __init__(self, model: ModelWrapper, tracer: sdk_trace.Tracer) -> None:
         self._model = model
         self._tracer = tracer
 
-    def _safe_lookup_model(self, model_name: str = MODEL_BASENAME) -> ModelWrapper:
```

Comment:
Will confirm in Slack, but I can't think of any reason why we need to preserve the match on name given that we hardcode [here](https://github.com/basetenlabs/truss/blob/42b6e752a9e1ec39927e78ddcdadd588e113c062/truss/templates/server/model_wrapper.py?plain=1#L40) and in [beefeater](https://github.com/basetenlabs/baseten/blob/fe2a2c7ad10d8124d38284171439e2cfa103b5cc/go/beefeater/middlewares/predict_utils.go?plain=1#L27-L28)